### DISCUSSION TITLE
Configure Conbee II correctly

### DISCUSSION THREAD

**Discussion by danielgut8330** [Koenkk/zigbee2mqtt#11038] [General]
Hello, I've been trying to get the Conbee II to run with the Zigbee2mqtt for days. Unfortunately without success.
I installed the Mosquito Broker. I also created an MQTT user and specified it in the configuration.
My configuration looks like this:
data_path: /config/zigbee2mqtt
external_converters: []
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  user: mqtt.user
  server: mqtt://core-mosquitto:1883
  passwort: mqtt.passwort
serial:
  port: /dev/ttyACM0
advanced:
  log_level: warn
  pan_id: 6754
  channel: 11
  network_key:
    - 1
    - 3
    - 5
    - 7
    - 9
    - 11
    - 13
    - 15
    - 0
    - 2
    - 4
    - 6
    - 8
    - 10
    - 12
    - 13
  availability_blocklist: []
  availability_passlist: []
device_options: {}
blocklist: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental: {}
availability: false
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false

In the log I always have the following error messages, which unfortunately mean nothing to me:
Zigbee2MQTT:error 2022-01-26 19:11:39: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-01-26 19:11:39: Failed to start zigbee
Zigbee2MQTT:error 2022-01-26 19:11:39: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-01-26 19:11:39: Exiting...
Zigbee2MQTT:error 2022-01-26 19:11:39: Error: Error while opening serialport 'Error: Error: Resource busy, cannot open /dev/ttyACM0'

Can someone help me here that not only 500:bad Gaitway appears in the user interface? that I can get the Zigbee2mqtt to run. would be very happy

**Comment by kdutton1964** [comment]
This from supported adapters in the docs. It should help.
USB connected adapter Conbee 2
If Zigbee2MQTT fails to start, try adding the following to your configuration.yaml

serial:
  adapter: deconz

Place after the serial port config. 
https://www.zigbee2mqtt.io/guide/adapters/#recommended

**Reply by danielgut8330** [reply]
Thank you for your answer.
I added the adapter: deconz. 
Unfortunately without success.my configuration file now looks like this:

data_path: /config/zigbee2mqtt
external_converters: []
devices: devices.yaml
groups: groups.yaml
home assistant: true
permit_join: false
qtt:
  base_topic: zigbee2mqtt
  user: mqtt.user
  server: mqtt://core-mosquitto:1883
  password: mqtt.password
serial:
  port: /dev/ttyACM0
  adapter: deconz
advanced:
  log_level: warn
  pan_id: 6754
  channels: 11
  network_key:
    - 1
    - 3
    - 5
    - 7
    - 9
    - 11
    - 13
    - 15
    - 0
    - 2
    - 4
    - 6
    - 8th
    - 10
    - 12
    - 13
  availability_blocklist: []
  availability_passlist: []
device_options: {}
block list: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental: {}
availability: false
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false

the error messages are still the same and the UI still says 502: Bad Gateway.
what else can I do?

**Reply by Fillll** [reply]
@kdutton1964, thank you! That works for me.

@danielgut8330, I do have different port of the device, probably similar issue for you:
```yaml
serial:
  adapter: deconz
  port: >-
    /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2411594-if00
```
